cmd/janus/server.go

```diff
@@ -39,10 +53,12 @@ func RunServer(cmd *cobra.Command, args []string) {
 
 	initLogger(globalConfig.LogLevel)
 	initTracing(globalConfig.Tracing)
-	storage := initStorage(globalConfig.Storage)
-	statsClient := initStatsdClient(globalConfig.Stats)
+	storage = initStorage(globalConfig.Storage)
+	statsClient = initStatsdClient(globalConfig.Stats)
 	defer statsClient.Close()
 
+	go startPubSubLoop(storage)
+
 	dsnURL, err := url.Parse(globalConfig.Database.DSN)
 	switch dsnURL.Scheme {
```

Comment:
[Not relate to this PR] - shouldn't the store strategy be on `pkg/store/store.go`?